packages/manager/apps/pci-object-storage/src/constants.ts

```diff
@@ -506,6 +501,8 @@ export const UNIVERSE = 'PublicCloud';
 export const SUB_UNIVERSE = 'storages';
 export const APP_NAME = 'objects_storage_container';
 
-export const STANDARD = 'STANDARD';
-export const HIGH_PERFORMANCE = 'HIGH_PERF';
-export const STANDARD_INFREQUENT_ACCESS = 'STANDARD_IA';
+export enum TReplicationStorageEnum {
```

Comment:
Please remove the `T` prefix as it's not a type (and the `enum` keyword)

```suggestion
export enum ReplicationStorageClass {
```